examples.md

```diff
@@ -217,7 +217,7 @@ If using `npm config` to retrieve the cache directory, ensure you run [actions/s
   with:
     path: |
       node_modules
-      */*/node_modules
+      **/*/node_modules
```

Comment:
Can this be simplified to just `path: **/node_modules`?

If I'm interpreting this correctly, the old version would match a top-level `node_modules` and all `node_modules` nested two folders deep.  The proposal here is to match all `node_modules`.

Reply:
Oh, you are totally right! fixed it so 👍